modules/apps/27-interchain-accounts/controller/keeper/handshake.go

```diff
@@ -52,7 +51,7 @@ func (k Keeper) OnChanOpenInit(
 
 	activeChannelID, found := k.GetOpenActiveChannel(ctx, connectionHops[0], portID)
 	if found {
-		return sdkerrors.Wrapf(porttypes.ErrInvalidPort, "existing active channel %s for portID %s", activeChannelID, portID)
+		return sdkerrors.Wrapf(icatypes.ErrActiveChannelAlreadySet, "existing active channel %s for portID %s", activeChannelID, portID)
```

Comment:
Should we include the connectionID in the error msg?

Reply:
not necessary since the active channel is returned, but might be nice to have